eng/Version.Details.xml

```diff
@@ -38,19 +38,19 @@
       <Uri>https://github.com/aspnet/AspNetCore</Uri>
       <Sha>397f924e8da45975d38f5d2c2a1e097399365eba</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Common.ItemTemplates" Version="5.0.0-alpha1.19401.1">
+    <Dependency Name="Microsoft.DotNet.Common.ItemTemplates" Version="5.0.0-alpha1.19401.1" CoherentParentDependency="Microsoft.Dotnet.Toolset.Internal">
```

Comment:
I am not sure this is right? This is for the templates themselves, that happen to be shipped out of the template engine repo, which is why the versions match. Not sure it is right to tie to the two of them together.

Reply:
The templates also flow into cli, which flows into toolset.  Since we want things to be coherent when we ship, we can simply remove the edge from templating->core-sdk by saying that the templating version should be taken from the version that flows into cli, then into toolset

![image](https://user-images.githubusercontent.com/8725170/62581203-5213a800-b85d-11e9-8889-350e461285dc.png)